### ISSUE TITLE
NoMethodError: undefined method `map'

### ISSUE THREAD

**Issue by floehopper** [freerange/mocha#443]
I noticed [this PR][1] in which they [tried to use v1.10.2][2] to fix the issue in #436, but ran into [another problem][3].

```
NoMethodError: undefined method `map' for #<String:0x000055f157c8de00>
    lib/mocha/parameters_matcher.rb:31:in `matchers'
    lib/mocha/parameters_matcher.rb:24:in `mocha_inspect'
    lib/mocha/invocation.rb:29:in `block in call'
    lib/mocha/multiple_yields.rb:11:in `block in each'
    lib/mocha/multiple_yields.rb:10:in `each'
    lib/mocha/multiple_yields.rb:10:in `each'
    lib/mocha/invocation.rb:23:in `call'
    lib/mocha/expectation.rb:571:in `invoke'
    lib/mocha/mock.rb:315:in `method_missing'
```

[1]: https://github.com/apiology/quality/issues/121
[2]: https://github.com/apiology/quality/commit/ec9b4ce0f5bb7f0db84316c6bb4a4d57bb924f4f
[3]: https://circleci.com/gh/apiology/quality/351

**Comment by floehopper** [comment]
I'm currently investigating.

**Comment by floehopper** [comment]
See possible fix in #444

**Comment by floehopper** [comment]
Fixed in #444.
